instructor/patch.py

```diff
@@ -128,37 +236,46 @@ async def new_create_async(
         )
         return response
 
-    @wraps(func)
+    @wraps(func)  # type: ignore
     def new_create_sync(
-        response_model: type[T_Model] = None,
-        validation_context: dict = None,
+        response_model: type[T_Model] | None = None,
+        validation_context: dict[str, Any] | None = None,
+        context: dict[str, Any] | None = None,
         max_retries: int = 1,
         strict: bool = True,
         *args: T_ParamSpec.args,
         **kwargs: T_ParamSpec.kwargs,
     ) -> T_Model:
+        context = handle_context(context, validation_context)
+
         response_model, new_kwargs = handle_response_model(
             response_model=response_model, mode=mode, **kwargs
         )
+
+        if "messages" in new_kwargs:
+            new_kwargs["messages"] = handle_templating(new_kwargs["messages"], context)
+        elif "message" in new_kwargs and "chat_history" in new_kwargs:
```

Comment:
@jxnl , Cohere client seems to have a separate async method in its definition, that's why we only modify the sync here.

Thoughts on this approach to help support cohere messages?